mkt/account/urls.py

```diff
@@ -26,6 +26,10 @@
         name='account-settings'),
     url(r'^shelves/$', FeedShelfViewSet.as_view(
         {'get': 'mine'}), name='feedshelves-mine'),
+    url('^groups/(?P<pk>[^/]+)/',
+        GroupsViewSet.as_view({'get': 'list', 'post': 'create',
+                               'delete': 'destroy'}),
+        name='account-groups'),
```

Comment:
You need a `$` at the end of that pattern. I'm not a super fan or the manually registered `ViewSet` since it has 3 actions (I usually use a `Route` whenever I can to avoid that), but I guess it's fine.